docs/userguide/basics.md

```diff
@@ -5,14 +5,32 @@ presentation is the width and height of the resulting PDF pages, which also affe
 aspect ratio. The default size if `1024x768`, which corresponds to aspect ratio `4:3`.
 
 Here is an example of creating a new presentation:
+
 ```python
 import elsie
+
 slides = elsie.SlideDeck(width=1024, height=768)
 ```
+
 Once you have a `SlideDeck` object, you can create new slides from it.
 
+## Backends
+*Elsie* needs a `Backend` to render slides to PDF. Currently, two backends are implemented:
+
+- [`InkscapeBackend`](elsie.render.backends.svg.backend.InkscapeBackend) is the default backend. This
+  backend uses `Inkscape` to convert slides to PDF, which means that you need to have `Inkscape`
+  installed on your computer.
+- [`CairoBackend`](elsie.render.backends.cairo.backend.CairoBackend) is an experimental backend
+  that uses the [Cairo](https://www.cairographics.org/) library to render slides. It supports most
+  of *Elsie* features, it should produce very similar output to the Inkscape backend and most
+  importantly, you do not need `Inkscape` installed for it to work. However, it is currently
+  experimental and not tested as thoroughly as the Inkscape backend.
+
+To use a specific backend, simply pass it to `SlideDeck` using the `backend` parameter.
```

Comment:
It is not clear to me from the doc if I should pass a class or an instance of a class. Maybe an example can help?